arch/arm/src/nrf52/hardware/nrf52_clock.h

```diff
@@ -118,9 +118,11 @@
 /* HFCLKSTAT Register */
 
 #define CLOCK_HFCLKSTAT_SRC_SHIFT       (0)       /* Bit 0: Source of HFCLK */
+
```

Comment:
These were introduced because otherwise nxstyle complains that right hand comments are not aligned. I could make all comments be aligned but there are a few longer lines which would put them very far to the right.

Reply:
OK, now I understand. But I think this is more like a nxstyle-hack than the correct solution and we should align the comments properly.

Reply:
Ok, I'll align comments.